text/trace/0000-error_flagging.md

```diff
@@ -0,0 +1,86 @@
+# Error Flagging with Status Codes
+This proposal adds two status codes explicitly for use as overrides by the end user, and proposes a canonical mapping of semantic conventions to status codes. This clarifies how error reporting should work in OpenTelemetry.
+ 
+## Motivation
+Error reporting is a fundamental use case for distributed tracing. While we prefer that error flagging occurs within analysis tools, and not within instrumentation plugins, a number of currently supported analysis tools and protocols rely on the existence of an explicit error flag reported from instrumentation. In OpenTelmetry, the error flag is called "status codes."
+ 
+However, there is confusion over the mapping of semantic conventions to status codes, and concern over the subjective nature of errors. Which network failures count as an error? Are 404s an error? The answer is dependent on the situation.
+ 
+There is one major exception. Both application developers and operators have a deep understanding of what constitutes an error in their system. OpenTelemetry must provide a way for these users to control error flagging, and explicitly indicate both when a span should and should not be counted as an error.
+A second exception is supporting analysis tools which require explicit error flagging in the data which they receive. In this case, an operator must be able to apply an error flagging schema at some point during the OTLP data processing pipeline.
```

Comment:
Maybe the reference to OTLP is misplaced but I understood this as "SDK exports as OTLP to collector, collector applies the error flagging schema, exports via Jaeger to backend or via OTLP to another collector."

Reply:
Yeah I meant what @Oberon00 said. I mentioned  OTLP because it will be the only format which has these override features, but I can change it if it is confusing. I'm also realizing that we don't define our "data processing pipeline" anywhere, so I see how that term could be confusing, so I'll add an explanation.